packages/remix-cloudflare/index.ts

```diff
@@ -1,6 +1,10 @@
 import "./globals";
 
-export { createCloudflareKVSessionStorage } from "./sessions/cloudflareKVSessionStorage";
+export {
+  createWorkersKVSessionStorage,
+  // TODO: Deprecate createCloudflareKVSessionStorage
+  createWorkersKVSessionStorage as createCloudflareKVSessionStorage,
```

Comment:
I think it makes sense to have `createCloudflareKVSessionStorage` be a wrapper function for `createWorkersKVSessionStorage` that just `console.warn`s before delegating:

```ts
function deprecate<T extends Function>(fn: T, message: string): T {
    return <any>function(...args) {
        console.warn(message)
        return fn(...args)
    };
}

const createCloudflareKVSessionStorage = deprecate(createWorkersKVSessionStorage, "`createCloudflareKVSessionStorage` is deprecated. Use `createWorkersKVSessionStorage` instead.")
```

Reply:
Adding a `@deprecated` annotation to it would be great as well I think

```suggestion
function warn<T extends Function>(fn: T, message: string): T {
    return <any>function(...args) {
        console.warn(message)
        return fn(...args)
    };
}

/**
 * @deprecated Use `createWorkersKVSessionStorage` instead.
 */
const createCloudflareKVSessionStorage = warn(
  createWorkersKVSessionStorage,
  "`createCloudflareKVSessionStorage` is deprecated. Use `createWorkersKVSessionStorage` instead.",
)
```